libvirt/tests/src/cpu/cpu_topology_die_id.py

```
@@ -0,0 +1,198 @@
+
+    return results
+
+
+def run(test, params, env):
+    """
+    Test start VM with hyper-v related features
+    """
+    vm_name = params.get('main_vm')
+    vm = env.get_vm(vm_name)
+
+    vm_attrs = eval(params.get('vm_attrs', '{}'))
```

Ruff (0.12.2)

Use of possibly insecure function; consider using `ast.literal_eval`

(S307)